Library/Homebrew/extend/kernel.rb

```diff
@@ -64,6 +64,7 @@ def oh1(title, truncate: :auto)
   def opoo(message)
     Tty.with($stderr) do |stderr|
       stderr.puts Formatter.warning(message, label: "Warning")
+      puts "::warning::#{message}" if ENV["GITHUB_ACTIONS"]
```

Comment:
We should probably use the `GitHubActions::Annotations` class here.

Also, would be nice if we could show these annotations at the source location of the caller...

Reply:
https://github.com/Homebrew/brew/blob/21e130056a009e48e863dc9969b9008b4626f1fe/Library/Homebrew/utils/github/actions.rb#L39

Reply:
> > Not sure I understand what this means.
> 
> I think it means: please make this show where it happened

Yes, this, using https://github.com/Homebrew/brew/blob/21e130056a009e48e863dc9969b9008b4626f1fe/Library/Homebrew/utils/github/actions.rb#L68-L73

Though I guess you would need the source location of the caller, which I don't know how to get from the callee.

Reply:
Thanks for explaining, all; searched for `GitHubActions` and `Annotations` and found neither 😅. Updated accordingly.

> Though I guess you would need the source location of the caller, which I don't know how to get from the callee.

I don't think this makes sense for most `opoo` messages. Does it mean anything to an end-user that it came from e.g. `reinstall.rb` rather than `install.rb`?

Where I think this does make sense is for deprecations/disables so I've passed them there.